### ISSUE TITLE
the published version isn't the latest from github

### ISSUE THREAD

**Issue by coyotebringsfire** [stevemacn/PageRank#8]
When I install pagerank-js via npm, node_modules/pagerank-js/lib/pagerank.js is missing the code which checks if a node has no incoming nodes, before checking for the number of incoming nodes. The code was merged into master, but that isn't the version that's installed by npm 

``` javascript
        if( this.incomingNodes[b] ) {
            for ( a=0; a<this.incomingNodes[b].length; a++) {
                prob = this.probabilityNodes[ this.incomingNodes[b][a] ];
                ct = this.outgoingNodes[ this.incomingNodes[b][a] ].length;
                sum += (prob/ct) ;
            }
        }
```

https://github.com/stevemacn/PageRank/blob/master/lib/pagerank.js#L102-L108

**Comment by coyotebringsfire** [comment]
if I install the latest from github, the right code is installed

```
$ npm install git://github.com/stevemacn/PageRank.git
```

**Comment by stevemacn** [comment]
Fixed NPM version bumped to 3.0
